tests/test_router_bgp.rb

```diff
@@ -20,17 +20,38 @@
 require_relative 'ciscotest'
 require_relative '../lib/cisco_node_utils/bgp'
 
+XR_NOT_SUPPORTED     = 'Not supported in IOS XR'
```

Comment:
Are there really so many commands that work in NXOS but not in IOS XR? I though XR was supposed to be the gold standard for BGP.